iOS_SDK/OneSignalSDK/Source/NSString+OneSignal.h

```diff
@@ -18,7 +18,7 @@
 - (NSString *)supportedFileExtension;
 
 // returns a lower case hex representation of the data
-+ (NSString *)hexStringFromData:(NSData *)data;
++ (nullable NSString *)hexStringFromData:(nonnull NSData *)data;
```

Comment:
can we use _Nullable _Nonnull only for convention purposes?

Reply:
We do have a mixture right now, I think the lower case is cleaner and easier to read. These lowercase versions seem to be exactly the same function too.
```
The nullable and nonnull annotations are simplified forms of the _Nullable and _Nonnull annotations
```
https://developer.apple.com/documentation/swift/objective-c_and_c_code_customization/designating_nullability_in_objective-c_apis